rust/benches/file_backend.rs

```diff
@@ -354,6 +359,7 @@ fn file_backend_benchmark(
             let chunk_size = *chunk_size;
             let threads = *threads;
             b.iter_custom(|iterations| {
+                init_file_fn();
```

Comment:
Shouldn't this also work?
```suggestion
            init_file_fn();
            b.iter_custom(|iterations| { 
```